package.json

```diff
@@ -3,17 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-free": "5.15.3",
     "@polkadot/api": "^6.1.1",
-    "@types/node": "^12.0.0",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "bootstrap": "^5.1.1",
+    "bootstrap": "4.6.0",
+    "classnames": "2.3.1",
+    "moment": "2.29.1",
```

Comment:
have u considered using `date-fns` instead of `moment`? it's much smaller

Reply:
Oh, i will check that, if its ligther better, as I'am just using a date formatter in the one row label, will consider it for next changes.